Core/PixelEvent.swift

```diff
@@ -1061,7 +1061,7 @@ extension Pixel.Event {
         case .networkProtectionActivationRequestFailed: return "m_netp_network_extension_error_activation_request_failed"
         case .networkProtectionFailedToStartTunnel: return "m_netp_failed_to_start_tunnel"
         case .networkProtectionDisconnected: return "m_netp_vpn_disconnect"
-        case .networkProtectionNoAuthTokenFoundError: return "m_netp_no_auth_token_found_error"
+        case .networkProtectionNoAccessTokenFoundError: return "m_netp_no_access_token_found_error"
```

Comment:
We want to rename this to separate ourselves from the noisy version that is no longer reliable. I'm taking this as an opportunity to fix the language, where we now call it an access token, not auth token.